packages/projection/src/operators/certificates/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './withCertificates';
 export * from './withStakeKeys';
 export * from './withStakePools';
+export * from './withAdaHandle';
```

Comment:
Please create a new `operators/misc` directory for this
```suggestion
export * from './misc';
```